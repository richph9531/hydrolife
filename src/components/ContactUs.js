import React from 'react';

import classes from '../styles/ContactUs.module.css';

const ContactUs = () => (
  <div className={classes.ContactUs}>
  <h1>Contact Us</h1>
</div>
);

export default ContactUs;
