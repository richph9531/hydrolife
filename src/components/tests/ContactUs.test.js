import React from 'react';

import { render } from '@testing-library/react';
import ContactUs from '../ContactUs';

test('Contact Us renders correctly', () => {
  const view = render(<ContactUs />);
  expect(view.baseElement).toMatchSnapshot();
});
