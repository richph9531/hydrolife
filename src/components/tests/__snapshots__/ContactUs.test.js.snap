// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Us renders correctly 1`] = `
<body>
  <div>
    <div
      class="ContactUs"
    >
      <h1>
        Contact Us
      </h1>
      <a
        href="tel:0878203953"
        style="color: rgb(255, 255, 255);"
      >
        087 820 3953
      </a>
    </div>
  </div>
</body>
`;
